learn/teams/overview.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Overview"
+id: ""
+---
+---
+
+**Teams** are groups of organization members that reflect your organization and organize your group’s work. Teams are designed and intended to be administered by one or more members. On a team's page, you can 
```

Comment:
Needs rephrasing

Reply:
Working on it.

Reply:
You can review it now. @prasanth-reddy-wavemaker-com